community/kernel/src/main/java/org/neo4j/kernel/DatabaseAvailability.java

```diff
@@ -63,18 +74,18 @@ public void stop()
     {
         // Database is no longer available for use
         // Deny beginning new transactions
-        availabilityGuard.deny(this);
+        availabilityGuard.deny( this );
 
         // Await transactions stopped
-        awaitNoTransactionsOr( 10_000  /* ms */);
+        awaitTransactionsClosedWithinTimeout();
     }
 
-    private void awaitNoTransactionsOr( int orUntilDeadline )
+    private void awaitTransactionsClosedWithinTimeout()
     {
-        long deadline = SYSTEM_CLOCK.currentTimeMillis() + orUntilDeadline;
-        while ( transactionMonitor.getNumberOfActiveTransactions() > 0 && SYSTEM_CLOCK.currentTimeMillis() < deadline)
+        long deadline = SYSTEM_CLOCK.currentTimeMillis() + awaitActiveTransactionDeadlineMillis;
+        while ( transactionMonitor.getNumberOfActiveTransactions() > 0 && SYSTEM_CLOCK.currentTimeMillis() < deadline )
         {
-            Thread.yield();
+            LockSupport.parkNanos( 10_000_1000 );
```

Comment:
Dangit, mistyped that one. Also I use park instead of sleep since the try-catch is always so ugly to have... and it shouldn't be any functional difference between them afaik. Any reason park would be bad?